# sentences-per-line/one

ðŸ’¼ This rule is enabled in the âœ… `recommended` config.

ðŸ”§ This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

This rule enforces that each at most one sentence appears per line in Markdown files.
If two or more sentences appear on the same line, they will be split onto separate lines.

## Examples

Example of **incorrect** code for this rule:

```md
First sentence.
Second sentence
```

Example of **correct** code for this rule:

```md
First sentence.
Second sentence
```
